addons/sales_team/tests/test_sales_team_membership.py

```diff
@@ -276,6 +276,29 @@ def test_memberships_sync(self):
             added.write({'crm_team_id': sales_team_1.id})
             self.new_team.flush()
 
+    def test_sales_team_member_search(self):
+        """ when a user is added to the sales team and then later removed
+        from the team, the entry in the crm.team.membership table is kept
+        archived.
+
+        then when a search is triggered on the member_ids field in crm.team
+        it is currently returning the archived records also. this test will
+        ensure that the search wont return archived record.
```

Comment:
Would not talk about 'now' and 'then' as it's not relevant in the long run. Would just keep the second sentence here.